src/pages/FieldVisibilitySettingsPage/FieldVisibilitySettingsPage.tsx

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import {
+  IonBackButton,
+  IonButtons,
+  IonContent,
+  IonHeader,
+  IonItem,
+  IonLabel,
+  IonList,
+  IonPage,
+  IonTitle,
+} from "@ionic/react";
+import paths from "../../paths";
+import { TEMPLATES } from "../../api";
+import ThemedToolbar from "../../components/ThemedToolbar/ThemedToolbar";
+import SlotSelectorModal from "../../components/SlotSelectorModal/SlotSelectorModal";
+import { useStore } from "../../Store";
+import Pluralize from "../../components/Pluralize/Pluralize";
+
+interface TemplateItemLabelProps {
+  template: string;
+}
+const TemplateItemLabel: React.FC<TemplateItemLabelProps> = ({ template }) => {
+  const { getHiddenSlotsForSchemaClass } = useStore();
+  const templateInfo = TEMPLATES[template];
+  const hiddenSlots = getHiddenSlotsForSchemaClass(templateInfo.schemaClass);
+  return (
+    <IonLabel>
+      <h3>{templateInfo.displayName}</h3>
+      <p>
+        {hiddenSlots === undefined ? (
+          "Not customized"
+        ) : (
+          <>
+            <Pluralize count={hiddenSlots.length} singular="field" showCount />{" "}
+            hidden
+          </>
+        )}
+      </p>
+    </IonLabel>
+  );
+};
+
+const FieldVisibilitySettingsPage: React.FC = () => {
+  const [selectedTemplate, setSelectedTemplate] = React.useState<
+    string | undefined
+  >(undefined);
+
+  return (
+    <IonPage>
+      <IonHeader>
+        <ThemedToolbar>
+          <IonButtons slot="start">
+            <IonBackButton defaultHref={paths.settings} />
+          </IonButtons>
+          <IonTitle>Field Visibility</IonTitle>
+        </ThemedToolbar>
+      </IonHeader>
+      <IonContent>
+        <p className="ion-padding">
+          Select a template to customize which fields are visible by default.
```

Comment:
This is another place where I am a little confused about the phrase "by default".

Assuming I understand the user-facing behavior, I think I'd find something like this to be more clear:

```diff
- Select a template to customize which fields are visible by default.
+ Select a template to customize which fields will be visible when using that template.
```